test/jdk/com/sun/management/HotSpotDiagnosticMXBean/DumpThreads.java

```diff
@@ -684,6 +692,16 @@ private boolean contains(List<String> lines, String text) {
                 .anyMatch(l -> l.contains(text));
     }
 
+    /**
```

Comment:
In the contains methods, it comment says "contain the given text", it should "containing the given text".